packages/core/src/modules/dids/domain/DidDocument.ts

```diff
@@ -18,6 +18,7 @@ type DidPurpose =
   | 'assertionMethod'
   | 'capabilityInvocation'
   | 'capabilityDelegation'
+  | 'verificationMethod'
```

Comment:
verificationMethod is not a purpose, but a generic set that holds keys for other purposes. 

```suggestion
```

Reply:
Created separate type for did verification methods